tests/cases/fourslash/codeFixInferFromUsagePropertyAccessJS.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+// @checkJs: true
+// @noImplicitAny: true

```

Comment:
I think this shouldn't be necessary because we should make it a suggestion diagnostic anyway, if we don't already.

Reply:
Either noImplicitAny isn't triggered as a suggestion (perhaps just for JS?), or the test is only seeing codefixes from errors. Any idea which? I don't know how to tell what gets triggered as a suggestion, do you?

Reply:
`verify.getSuggestionDiagnostics` should allow you to test that.

Reply:
There is one suggestion, but it's "'numeric' is declared but its value is never read.". I don't think noImplicitAny is leading to a infer-from-usage suggestion. It probably should someday, but I'd like to improve the precision and recall of it first.

Reply:
Filed #27624